specification/quantum/data-plane/readme.python.md

```diff
@@ -0,0 +1,21 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: Microsoft.Quantum
+  package-name: quantum
+  package-version: 2019-11-04-preview
```

Comment:
This arg is intended for package versionning (1.0.0, etc.). You could start with `1.0.0b1` for instance